var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -39,92 +40,112 @@ class Neuron(Package):
     version('7.3', '993e539cb8bf102ca52e9fefd644ab61')
     version('7.2', '5486709b6366add932e3a6d141c4f7ad')
 
-    variant('binary',        default=False, description="Create special as a binary instead of shell script")
-    variant('coreneuron',    default=True,  description="Patch hh.mod for CoreNEURON compatibility")
-    variant('cross-compile', default=False, description='Build for cross-compile environment')
-    variant('debug',         default=False, description='Build debug with O0')
-    variant('mpi',           default=True,  description='Enable MPI parallelism')
-    variant('multisend',     default=True,  description="Enable multi-send spike exchange")
-    variant('profile',       default=False, description="Enable Tau profiling")
-    variant('python',        default=True,  description='Enable python')
-    variant('shared',        default=True,  description='Build shared libraries')
-    variant('pysetup',       default=True,  description="Build Python module with setup.py")
-    variant('rx3d',          default=False, description="Enable cython translated 3-d rxd. Depends on pysetup")
+    variant('cmake',      default=False, description="Build NEURON using cmake")
+    variant('binary',     default=False, description="Create special as a binary instead of shell script")
+    variant('coreneuron', default=False, description="Patch hh.mod for CoreNEURON compatibility")
+    variant('cross-compile',  default=False, description='Build for cross-compile environment')
+    variant('debug',          default=False, description='Build with flags -g -O0')
+    variant('discrete-event-observer', default=True,  description='Enable Observer to be a subclass of DiscreteEvent')
+    variant('interviews', default=False, description='Enable GUI with INTERVIEWS')
+    variant('legacy-fr',  default=True,  description='Use original faraday, R, etc. instead of 2019 nist constants')
+    variant('mech-dll-style', default=True,  description='Dynamically load nrnmech shared library')
+    variant('memacs',     default=True,  description='Enable use of memacs')
+    variant('mpi',        default=True,  description='Enable MPI parallelism')
+    variant('multisend',  default=True,  description="Enable multi-send spike exchange")
+    variant('profile',    default=False, description="Enable Tau profiling")
+    variant('pysetup',    default=True,  description="Build Python module with setup.py")
+    variant('python',     default=True,  description='Enable python')
```

Comment:
I would just combine above two options as python.

Reply:
`pysetup` is only for the `autotools` build, right? With `cmake` the `python` package is built by default. Also, regarding the `dynamic python version support`, I guess that there is no usage in spack?